spec/models/contact_spec.rb

```diff
@@ -103,6 +103,19 @@
       end
     end
 
+    describe "mutual" do
+      it "returns contacts with sharing true and receiving true" do
+        expect {
+          alice.contacts.create!(receiving: true, sharing: true, person: FactoryGirl.build(:person))
```

Comment:
You could maybe test count == 1 right after this line, to be sure that mutual == sharing && receiving, and then add the other contacts and check again that the count didn't change.